docs/show-me.md

```diff
@@ -0,0 +1,31 @@
+# Show Me
+
+## Create Routes and Router
+
+In this clip, we're going to create (2) top level routes `home` and `settings`. Then we're going to add (2) additional routes under `settings`. Finally, we will create and export our `router`.
+
+![Create Routes and Router](/create-routes.gif)
+
+## Installing the Router
+
+In this clip, we take the router we install the router and setup the Type Safety in components with the `Register` interface. Read more about [updating the registered router](/getting-started#update-registered-router).
+
+![Installing the Router](/install-router.gif)
+
+## Navigating with Router Push
+
+In this clip, we call `useRouter` in setup of `HomeView` and call `router.push` when the button is clicked. Notice how all (4) of our routes show up in autocomplete. Typescript will throw an error if I use a value other than a valid route key or a fully formed external URL. Read more about [router push](/core-concepts/navigating#push).
+
+![Navigating with Router Push](/router-push.gif)
+
+## RouterView and RouterLink
+
+In this clip, we add `RouterView` component to `SettingsView`, since it has children. We also add a couple `RouterLink` components to the navbar to give the user links to those child pages. Notice how the `to` callback syntax is also type safe and autocompletes all possible routes for us.
```

Comment:
```suggestion
We need to add `RouterView` component to `SettingsView`, since it has children. We'll also add a couple `RouterLink` components to the navbar to give the user links to those child pages. Notice how the `to` callback syntax is also type safe and autocompletes all possible routes for us.
```